frontend/src/v5/services/api/default.ts

```diff
@@ -50,7 +47,6 @@ axios.interceptors.response.use(
 				case 403:
 					// eslint-disable-next-line no-param-reassign
 					error.handled = true;
-					dispatch(push('/'));
```

Comment:
What's the reason for removing this?

Reply:
This whole file was copied from a similar v4 file. The problem, however, is a circular dependency. Removing this line, the circular dependency can be broken